### PR TITLE
Weather api

### PR DESCRIPTION
**Pull request by IamTheCry**

### CODE REVIEW

File: server/apis/weather.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const request = require('request-promise');
+
+const weather = {
+
+  getWeather: (loc) => {
+    let crd = JSON.parse(loc)
```
Comment (jlei523) [inline server/apis/weather.js]: Suggest spelling out coordinates as a variable name for clarity

File: server/router.js [hunk 2]
```diff
@@ -25,11 +25,16 @@ router.post('/saved', (req, res) => {
 });
 
 router.post('/voice', (req, res) => {
-  console.log('I got your voice object!', req.body);
+  console.log('I got your voice object!',typeof req.body.loc, req.body);
```
Comment (jlei523) [inline server/router.js]: Do you need this change? If not, we should comment it out so we can clear out some console logs.

Comment (IamTheCry) [inline server/router.js]: Do not need, it slipped through my console log purge.

File: server/index.js [hunk 3]
```diff
@@ -19,7 +19,8 @@ app.use(express.static(__dirname + '/../react-client/dist'));
 //how to use cleverbot
 	// apis.cbot.getResponse('who is barack obama')
 	// .then((data) => console.log(data));
-
+apis.weather.getWeather('{"lat":"37.7837039","lon":"-122.4091297"}')
```
Comment (mttrys) [inline server/index.js]: If this is a test? if not comment out this line of code and create a generic api call that accept generic coord arguments

Comment (IamTheCry) [inline server/index.js]: Yes this is a test.

**Pull request merged**